raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -300,15 +308,13 @@ contract TokenNetwork is Utils {
     )
         external
```

Comment:
this is missing the `isOpen` modifier

Reply:
Additionally, shouldn't we move the checks `require(channel_identifier == getChannelIdentifier(participant, partner));` to the modifiers? (I didn't check the other PRs for this)

Reply:
This is for `setTotalWithdraw`. There is an equivalent:
```
// Channel must be open
require(channel.state == 1);
```
in the code. An equivalent to `isOpen`.

Totally agree on the modifiers. This is why I opened https://github.com/raiden-network/raiden-contracts/issues/197. We currently cannot be consistent with them from the reasons I wrote in the issue. I want to tackle the modifiers separately and not in this PR - are you ok with this?

Reply:
> are you ok with this?

as longs as the code is correct and it's fixed, yes